src/Smalot/PdfParser/Parser.php

```diff
@@ -262,6 +262,11 @@ protected function parseHeader(array $structure, ?Document $document): Header
      */
     protected function parseHeaderElement(?string $type, $value, ?Document $document)
     {
+        $valueIsEmpty = null == $value || '' == $value || false == $value;
+        if (('<<' === $type || '>>' === $type) && $valueIsEmpty)) {
```

Comment:
```suggestion
        if (('<<' === $type || '>>' === $type) && $valueIsEmpty) {
```
My bad, I missed a `)`.